### PR TITLE
Hub udpate for plugins FOSS Vol. 1 by Joey

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Proposed Changes

This PR updates the help and spec files on several plugins for the Hub launch.

### CODE REVIEW

File: vxstream_sandbox/plugin.spec.yaml [hunk 1]
```diff
@@ -6,17 +6,18 @@ title: VxStream Sandbox
...
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.payload-security.com/products/vxstream-sandbox
 tags:
 - malware
 - sandbox
 - malware analysis
 hub_tags:
-  use_cases: []
+  use_cases: [threat_detection_and_response, threat_hunting]
   keywords: [malware, sandbox, malware analysis]
```
Comment (mhofert-r7) [inline vxstream_sandbox/plugin.spec.yaml]: add `vxstream` so it's searchable by vender name

File: whois/help.md [hunk 2]
```diff
@@ -3,18 +3,16 @@
...
-* Feature 2
-* Feature 3
+* Query for an address
+* Query for a URL
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* None
```
Comment (mhofert-r7) [inline whois/help.md]: _This plugin does not contain any requirements._

File: wigle/plugin.spec.yaml [hunk 3]
```diff
@@ -3,23 +3,21 @@ extension: plugin
...
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.wigle.net/
 tags:
 - network
 - location
 - wireless
 hub_tags:
-  use_cases: []
+  use_cases: [threat_intel]
   keywords: [network, location, wireless]
```
Comment (mhofert-r7) [inline wigle/plugin.spec.yaml]: add `wigle`

File: wazuh_ossec/help.md [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-* Feature 3
+* Manage Wazuh agents
+* Investigate Wazuh logs
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* An administrative username and password
+* The url of your Wazuah deployment
```
Comment (jschipp-r7) [inline wazuh_ossec/help.md]: URL

File: whois/help.md [hunk 5]
```diff
@@ -3,18 +3,16 @@
 [WHOIS](https://en.wikipedia.org/wiki/WHOIS) is a query and response protocol that is widely used for querying databases that store the registered users
 or assignees of an Internet resource, such as a domain name, an IP address block, or an autonomous system, but is also used for a wider range of other information.
 
-This plugin uses the Linux Whois client to make requests.
+The WHOIS plugin enables address and domain lookups in the WHOIS databases.
```
Comment (jschipp-r7) [inline whois/help.md]: Trailing space

File: wigle/help.md [hunk 6]
```diff
@@ -1,19 +1,20 @@
...
-* Feature 3
+* Retrieve wireless network information
+* Update wireless information in WiGLE
+* Get statistics about regions or countries
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* A WiGLE api token
```
Comment (jschipp-r7) [inline wigle/help.md]: API

File: wigle/help.md [hunk 7]
```diff
@@ -1,19 +1,20 @@
...
+* The WiGLE api name to use
```
Comment (jschipp-r7) [inline wigle/help.md]: API

File: zeus_tracker/help.md [hunk 8]
```diff
@@ -1,18 +1,15 @@
...
-* Feature 1
-* Feature 2
-* Feature 3
+* NA - Depreciated
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* NA - Depreciated
```
Comment (jschipp-r7) [inline zeus_tracker/help.md]: Use the standard line here

_This plugin does not contain any requirements._

File: zeus_tracker/plugin.spec.yaml [hunk 9]
```diff
@@ -3,14 +3,15 @@ extension: plugin
 product: ["insightconnect"]
 name: zeus_tracker
 title: ZeuS Tracker
-description: ZeuS Command&Control Server Tracking
+description: The ZeuS Command&Control Server Tracking plugin
 version: 1.0.0
 vendor: rapid7
 support: community
-status: ["obsolete"]
+status: [deprecated]
```
Comment (jschipp-r7) [inline zeus_tracker/plugin.spec.yaml]: This plugin is obsolete, will need to revert this back

# ZeuS Tracker has been discontinued on Jul 8th, 2019

File: wazuh_ossec/plugin.spec.yaml [hunk 10]
```diff
@@ -12,12 +12,13 @@ status: []
...
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/wazuh_ossec
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://wazuh.com/
 tags:
 - ossec
 - wazuh
 - hids
 hub_tags:
-  use_cases: []
+  use_cases: [threat_intel, vulnerability_management, log_search,. detection_and_response, asset_containment, asset_inventory]
```
Comment (mhofert-r7) [inline wazuh_ossec/plugin.spec.yaml]: There is a period after `log_search,`

**Pull request merged**